client/my-sites/posts/main.jsx

```diff
@@ -34,21 +34,11 @@ import {
 import { getEditorNewPostPath } from 'state/ui/editor/selectors';
 import { warningNotice } from 'state/notices/actions';
 import {
-	getSiteAdminUrl,
 	getSiteSlug,
 	isJetpackSite,
-	siteHasMinimumJetpackVersion
 } from 'state/sites/selectors';
 
 const PostsMain = React.createClass( {
-	componentWillMount() {
-		this.setWarning( this.props );
-	},
-
-	componentWillReceiveProps( nextProps ) {
-		this.setWarning( nextProps );
```

Comment:
In my opinion, a lighter touch here may be a good thing. If this can be fixed by changing 1 line of code rather than adding ~40, that's probably a good thing 🙂